features/providers/deleting_applications.feature

```diff
@@ -1,34 +1,15 @@
 Feature: Deleting applications
 
   @javascript
```

Comment:
does feature work pass without the javascript tag?

Reply:
No, the features hang at the first step without the tag! 🤷‍♂️